mautic/mautic
# Ignore queue mode for send email to contact action

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/3828
| BC breaks? | 
| Deprecations? | 

This bugfix/feature has been sponsored by @Webmecanik

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
The same workflow like already merged feature https://github.com/mautic/mautic/pull/5414 
Just added option send email to contact immediately in queue mode.

![image](https://user-images.githubusercontent.com/462477/34921474-11f5bc4e-f983-11e7-8a2b-28e823ac4fe8.png)

#### Steps to test this PR:
Steps to test this PR:
1. Apply PR and got to Configurations > Emails > and Enable queue
2. Create form with Send email to Contact action with new Send immediately option set to No
3: Send form, email should go to queue (verify it in directory app\spool\default)
4. Then Go to form and edit action to Send immediately YES
5. Send from, email should send immediately without queue (again verify it in directory app\spool\default\ - new file shoudn't create)
6. Then change Configurations > Emails > and change Queue to immediately
7. Send form and email should send immediately (again verify it in directory app\spool\default\ - new file shoudn't create)
